js/glowingbear.js

```diff
@@ -197,7 +197,7 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
                         $rootScope.updateBufferBottom(true);
                         $rootScope.scrollWithBuffer();
                         bl.onscroll = _.debounce(function() {
-                            $rootScope.updateBufferBottom();
+                            $rootScope.updateBufferBottom($rootScope.bufferBottom);
```

Comment:
OK this is nonsense, it makes it (nearly) impossible to scroll away from the bottom. FFS.